dts/arm/renesas/ra/ra8/r7fa8d1bhecbd.dtsi

```diff
@@ -32,5 +33,23 @@
 			zephyr,memory-region = "SDRAM";
 			status = "disabled";
 		};
+
+		lcdif: display-controller@40342000 {
+			compatible = "renesas,ra-display";
+			reg = <0x40342000 0x1454>;
+			clocks = <&lcdclk MSTPC 4>;
+			status = "disabled";
+		};
+
+		mipi_dsi: dsihost@40346000 {
+			compatible = "renesas,ra-mipi-dsi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40346000 0x2000>;
+			interrupts = <72 1>, <73 1>, <74 1>, <75 1>, <76 1>, <77 1>;
+			interrupt-names = "sq0", "sq1", "vm", "rcv", "ferr", "ppi";
+			clocks = <&lcdclk MSTPC 10>;
+			status = "okay";
```

Comment:
Don't set nodes to `status = "okay"` at SOC level, keep them disabled here and enable at the board level